### PR TITLE
VACMS-3254: Update R&S term page views.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #3254

## Testing done
Visual / behat

## Screenshots
image
image
image

## QA steps
- As an administrator, go to `admin/config/system/taxonomy-entity-index/reindex`, check both entity types, and rebuild index.
- When indexing is complete, go to R&S page `/taxonomy/term/292`
- Visually verify that the title reads `All articles in: Sign in`
- Visually verify that rss link is not in footer
- Visually verify that page title sort order is a-z
- Visit `/node/6924` and visually verify that there aren't any link check messages related to term page links (3210)

### CODE REVIEW

**Review by oksana-c — CHANGES_REQUESTED** [review]
when I visit http://pr3321.ci.cms.va.gov/node/8222 I see "Sign in" - topics tag and "All Veterans" - Audience tag. 
Term content listing views are empty though:
* http://pr3321.ci.cms.va.gov/taxonomy/term/292
* http://pr3321.ci.cms.va.gov/taxonomy/term/268

please take a look, something may be missing in views config

**Comment by ethanteague** [comment]
Looks like you missed this QA step, @oksana-c : `As an administrator, go to admin/config/system/taxonomy-entity-index/reindex, check both entity types, and rebuild index.`
I just ran, and those pages are now populated
image
image

**Review by oksana-c — APPROVED** [review]
I sure did. Thanks @ethanteague - I can see it now ✅

File: config/sync/pathauto.pattern.taxonomy_terms.yml [hunk 1]
```diff
@@ -1,25 +0,0 @@
-uuid: 47be66c7-ece7-45e1-a63c-5c92ee31d9b3
```
Comment (oksana-c) [inline config/sync/pathauto.pattern.taxonomy_terms.yml:1]: something tells me this should be deleted

File: docroot/modules/custom/va_gov_backend/va_gov_backend.module [hunk 2]
```diff
@@ -452,10 +452,33 @@ function va_gov_backend_form_alter_submit(array &$form, FormStateInterface $form
...
+      'audience_beneficiaries',
+      'audience_non_beneficiaries',
+      'lc_categories',
+      'topics',
+    ];
+    $vocab = $term->get('vid')->getString();
+    if (in_array($vocab, $vocabs)) {
+
+      $variables['title']['#markup'] = t('All articles in:')
+      . ' ' . $variables['title']['#markup'];
```
Comment (oksana-c) [inline docroot/modules/custom/va_gov_backend/va_gov_backend.module:472]:
```suggestion
      $variables['title']['#markup'] = t('All articles in:') . ' ' . $variables['title']['#markup'];
```

**Review by oksana-c — CHANGES_REQUESTED** [review]
one of the new path patterns is deleted, let's bring it back. see comment

**Review by oksana-c — APPROVED** [review]
⚡

**Review by swirtSJW — APPROVED** [review]

**Pull request merged**